docs/reference/glossary.md

```diff
@@ -32,6 +32,16 @@ An image that has no parent is a **base image**.
 [boot2docker](http://boot2docker.io/) is a lightweight Linux distribution made
 specifically to run Docker containers. The boot2docker management tool for Mac and Windows was deprecated and replaced by [`docker-machine`](#machine) which you can install with the Docker Toolbox.
 
+## bridge network
+
+A bridge network is a Link Layer device (also called a Layer 2 device) which
```

Comment:
@aboch When I was just starting out I didn't have a canonical reference for what "bridge" meant in context of Docker, which is why I filed the original issue. If there's another canonical definition we can link to and then expand on (to cover the "in context of how Docker uses it" part), that would be fine.